travharv/executor.py

```diff
@@ -73,6 +79,18 @@ def assert_all_paths(self):
                             """
                         )
                         log.exception(e)
+                    finally:
+                        log.debug(
+                            f"Assertion path: {str(assertion_path)} asserted"
+                        )
+
+            # TODO figure out if the task_execution_report object
```

Comment:
sounds important enough to have an issue -- but maybe after 004 merge into py-sema

Reply:
mentioned in #55